[IMP] accounting: updated Adyen page (#2241)
odoo/documentation: content/applications/finance/payment_acquirers/adyen.rst

Adyen
=====

`Adyen <https://www.adyen.com/>`_ is a Dutch company that offers several online payment
possibilities.

.. seealso::
   - :ref:`payment_acquirers/add_new`
   - :doc:`../payment_acquirers`

Configuration

> =============

Reviewer: 2 things are bothering me here:

1) The `add_new` anchor is about the configuration, so why not keep it there?
2) Is there a reason to take off this `See also` from the end and put it above? 

But then, if it was decided to do this for all acquirers, this is fine I guess, but since it'd only be the case for Adyen, that seems strange though.

Author: I understand the logic better now. Indeed, it makes more sense. I changed it back down.

Adyen
=====

`Adyen <https://www.adyen.com/>`_ is a Dutch-based company that offers several online payment
`Adyen <https://www.adyen.com/>`_ is a Dutch company that offers several online payment
possibilities.

.. seealso::
   :ref:`payment_acquirers/add_new`

Configuration
=============

.. seealso::
   - :ref:`payment_acquirers/add_new`

> 

Reviewer: Configuration title still under the `seealso` ;)

Author: Done! :)

> `Adyen <https://www.adyen.com/>`_ is a Dutch-based company that offers several online payment
> possibilities.

Reviewer 2: ```suggestion
`Adyen <https://www.adyen.com/>`_ is a Dutch company that offers several online payment
possibilities.
```

   :align: center
   :alt: Configure a webhook.

There, in :menuselection:`General --> Server configuration --> URL`, enter your server address
followed by `/payment/adyen/notification`.

.. image:: adyen/adyen-webhook-url.png
   :align: center
   :alt: Enter the notification URL.

> Then continue in :menuselection:`Security --> HMAC Key --> Generate`. Be careful to copy it as you
> will not be allowed to get it later without generating a new one.

Reviewer 2: ```suggestion
Then continue to :menuselection:`Security --> HMAC Key --> Generate`. Be careful to copy the key as you
won't be allowed to do it later without generating a new one.
```

   :align: center
   :alt: Configure a webhook.

There, in :menuselection:`General --> Server configuration --> URL`, enter your server address
followed by `/payment/adyen/notification`.

.. image:: adyen/adyen-webhook-url.png
   :align: center
   :alt: Enter the notification URL.

Then enter :menuselection:`Security --> HMAC Key --> Generate`. Be careful to copy the key as you
will not be allowed to do it later without generating a new one.

> .. image:: adyen/adyen-hmac-key.png

Reviewer 3: there are still the two versions of the image file hmac-key: delete the original and rename the converted one

~~~~

To retrieve the URLs, log into your Adyen account, go to :menuselection:`Developers --> API URLs`.
Select one of the URLs listed next to :guilabel:`Classic Recurring API` as your
:guilabel:`Recurring API URL` and one listed next to :guilabel:`Checkout API` as your
:guilabel:`Checkout API URL`.

> .. image:: adyen/adyen-api-urls.png

Reviewer: The fact that the API names and urls are not aligned in this modified screenshot is triggering me, but not a problem I guess? :laughing:

Author: Fixed ;)

.. image:: adyen/adyen_allowed_origins.png
   :align: center

>    :alt: Allows payments originated from a specific domain.

Reviewer: As for API and Client Keys, this also slightly changed as it's now in the `Client Settings` part, which is new. There are some small visual changes too here ^^
